bin/prod

```diff
@@ -0,0 +1,53 @@
+PROD_ECR_REPO="public.ecr.aws/a5f0m6q3/syncm8_prod"
+
+# Build prod image.
+dockerBuild(){
+    bin/util logAndRun "docker build --pull --rm --target prod -t $PROD_ECR_REPO ."
+}
+
+# Push prod image to aws ecr.
+dockerPush(){
+    echo "Are you sure you want to push an image to PRODUCTION?????"
+    read ans
+    if [ $ans == "y" ]; then
```

Comment:
```suggestion
    read -p "Are you sure you want to push an image to PRODUCTION????? <y/N>"$'\n' ans
    if [[ $ans =~ [yY](es)* ]]; then
```

More UI friendly options. At least we should have what the user should enter after prompt.

![image](https://user-images.githubusercontent.com/12789302/142747525-39dc513c-60fc-408b-ac7e-a0a9eada73f9.png)

Reply:
I wanna make it really, hard to push to prod lol.
Will add a `echo not pushing to prod, to push to prod re-run this command and type 'y' at prompt` if it recognized not tho.

Reply:
lol I see. If that’s the case double confirmation or a long confirmation string are good alternatives too.